server/src/main/java/fr/uga/l3miage/tp4/models/ExaminerEntity.java

```diff
@@ -0,0 +1,22 @@
+package fr.uga.l3miage.tp4.models;
+
+import javax.persistence.DiscriminatorColumn;
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+public class ExaminerEntity extends UserEntity {
+
+    private String login;
+
+    @Column(length=6)
+    private String password;
```

Comment:
C'était sur login cette contrainte
```suggestion
    @Column(length=6)
    private String login;

    private String password;
```